src/components/shared/chart/Viewport.js

```diff
@@ -419,7 +419,7 @@ export const withChartViewport: WithChartViewport<*, *> =
       }
 
       zoomWithMouseWheel(
```

Comment:
Would you mind copying and pasting your excellent description of this problem here, so in the future we'll know why this event is not a React event and regress this behavior.

Reply:
Done!